src/engine/client/cl_download.cpp

```diff
@@ -294,44 +281,26 @@ void CL_WWWDownload()
 		*cls.downloadTempName = *cls.downloadName = 0;
 		Cvar_Set( "cl_downloadName", "" );
 
-		if ( !cls.bWWWDlDisconnected )
-		{
-			CL_AddReliableCommand( "wwwdl done" );
+		CL_AddReliableCommand( "wwwdl done" );
 
-			// tracking potential web redirects leading us to wrong checksum - only works in connected mode
-			if ( strlen( clc.redirectedList ) + strlen( cls.originalDownloadName ) + 1 >= sizeof( clc.redirectedList ) )
-			{
-				// just to be safe
-				Log::Warn( "redirectedList overflow (%s)\n", clc.redirectedList );
-			}
-			else
-			{
-				strcat( clc.redirectedList, "@" );
-				strcat( clc.redirectedList, cls.originalDownloadName );
-			}
-		}
-	}
-	else
-	{
-		if ( cls.bWWWDlDisconnected )
+		// tracking potential web redirects leading us to wrong checksum - only works in connected mode
+		if ( strlen( clc.redirectedList ) + strlen( cls.originalDownloadName ) + 1 >= sizeof( clc.redirectedList ) )
```

Comment:
this makes me discovering there is code to attempt to handle redirects… because of course redirects don't work! :smirk:

Reply:
There is not really any code to handle redirects. It's just a list of downloaded files.